plugins/mergemedia.py

```diff
@@ -0,0 +1,107 @@
+"""MergeMedia"""
+#  Copyright (C) 2020 BY USERGE-X
+#  All rights reserved.
+#
+#  Author: https://github.com/midnightmadwalk [TG: @midnightmadwalk]
+
+
+import os
+import re
+import codecs
+
+import shutil
+from pathlib import Path
+
+from userge import Message, userge
+from userge.plugins.misc.upload import upload
+from userge.utils import progress, runcmd
+from hachoir.stream.input import NullStreamError, InputStreamError 
+
+@userge.on_cmd(
+    "mergesave",
+    about={
+        "header": "save file for {tr}merge",
+        "usage": "{tr} reply to [media] for saving it ",
+    },
+)
+async def mergesave_(message: Message):
+    """mergesave"""
+    # saving files in a separate folder.
+    await message.edit("`downloading ...`")
+    try:
+        o_o = await message.client.download_media(
+          message=message.reply_to_message,
+          file_name='merge/',
+          progress=progress,
+          progress_args=(
+            message,
+            "`Saving for further merge !`"
+          ),
+        )
+    except AttributeError:
```

Comment:
Instead off attribute error you can also check if the media is downloadable
Like => if not message.photo or message.video:

Reply:
okay 💣🙃🙂